bleu/op-bacchus
# Integrate UI of event details with ticket real data

# Summary

- add new query to get tickets by event id
- use the query in event page details, so now the ticket listing are real
- create hooks to get event data by id and tickets by

# How to test

1. Go to Event > My events
2. Enter a event you own (if you don't have, create one in Event > New event)
- You should be able to see the ticket management section with a button to assign tickets
3. Assign some tickets
- These tickets should appear in the management section
